lib/core/CMakeLists.txt

```diff
@@ -79,6 +79,7 @@ target_link_libraries(
   PUBLIC
   nlohmann_json::nlohmann_json
   fmt::fmt
+  spdlog::spdlog_header_only # TODO: switch to spdlog::spdlog once shared lib is in externals
```

Comment:
RE TODO: We have an issue for this already, I assume?

Reply:
No, I'm just going to keep #6256 open